### ISSUE TITLE
Module name convention

### ISSUE THREAD

**Issue by clenk** [mitre/multiscanner#52]
Some modules have a `NAME` value that starts with an uppercase character, and others start with lowercase. This makes the output of `parse_reports` a little odd, as uppercase comes before lowercase in the sorting of module names. (e.g. "Yara" comes before "libmagic") Could we make the `NAME`s of the modules use consistent casing?

**Comment by Drewsif** [comment]
I agree, I think it was to originally follow the capitalization of the projects. We could also look into sorting without the caps.

**Comment by clenk** [comment]
`json` only lets you set sorting to be True or False. We could use [OrderedDicts](https://stackoverflow.com/questions/10844064/items-in-json-object-are-out-of-order-using-json-dumps), or use the `simplejson` library, which lets you pass in an `item_sort_key` to control how they get sorted. For instance, `lambda s: s.lower()`.
